docs/content/latest/yugabyte-platform/install-yugabyte-platform/install-software/kubernetes.md

```diff
@@ -130,15 +132,21 @@ Generated the kubeconfig file: /tmp/yugabyte-helm.conf
 
 4. Run the following `helm install` command to install Yugabyte Platform (`yugaware`) Helm chart:
 
-```sh
-helm install yw-test yugabytedb/yugaware --version 2.3.3 -n yb-platform --wait
-```
+    ```sh
+    helm install yw-test yugabytedb/yugaware --version 2.3.3 -n yb-platform --wait
+    ```
+
+5. Optionally, set the TLS version for Nginx frontend by using `ssl_protocols` operational directive in the Helm installation, as follows:
+
+    ```sh
+    helm install yw-test yugabytedb/yugaware --version 2.3.3 -n yb-platform --wait --set tls.sslProtocols="TLSv1.2 TLSv1.3"
```

Comment:
for nginx, TLS1.3 is supported. 
Please add two more parameters to nginx part.
ssl_protocols TLSv1.2 TLSv1.3; # Tell client which ciphers are available
ssl_prefer_server_ciphers on;
ssl_ciphers ECDH+AESGCM:ECDH+AES256:ECDH+AES128:DH+3DES:!ADH:!AECDH:!MD5; # this is customized and vary from customers security requirement. Some customers don't like 3DES.

Reply:
@chirag-yb @ssung-yugabyte Chirag, please confirm that you are okay with the changes that Stanley suggested

Reply:
@ssung-yugabyte since we don't support TLS v1.3 in the entire product I recommend not mentioning it anywhere. I would imaging enterprises keeping the same version across the entire platform and not assigning different versions to YB Nginx and YB DBs.

Reply:
@iSignal do we support setting ciphers from our helm charts?

Reply:
@chirag-yb Would you like me to merge now? Or wait for answer?

Reply:
@lizayugabyte Please merge it for now.